SUI/SUI/ContentView.swift

```diff
@@ -7,20 +7,244 @@
 
 import SwiftUI
 
+/// Экран настроек
 struct ContentView: View {
+    
+    private enum Constant {
+        static let wifiParam = ["Подключено", "Не подключено"]
+        static let onOfParam = ["Вкл.", "Выкл."]
+        static let avatarImageName = "galina"
+        static let avatarFrameWidth = 100
+        static let avatarFrameHeight = 100
+        static let accountName = "Galina"
+        static let accountNameOffset = -55
+        static let appleIDfontSize = 30
+        static let appleIDText = "Apple ID, iCloud, контент и покупки"
+        static let notifyXOffset = 120
+        static let fontSize = 20
+        static let picFrame = 30
+        static let cornerRadius = 5
+        static let notifyText = "4"
+        static let emptyString = ""
+        static let appleOffersText = "Предложения Apple ID"
+        static let settingText = "Настройки"
+        static let newiOSText = "iOS 16.2: уже доступно"
+        static let airplaneImageName = "airplane"
+        static let inFlightModeText = "Авиарежим"
+        static let wifiImageName = "wifi"
+        static let wifiPickerText = "Wi-Fi"
+        static let bluetoothImageName = "bluetooth"
+        static let bluetoothPickerText = "Bluetooth"
+        static let antennaImageName = "antenna.radiowaves.left.and.right"
+        static let cellularPickerText = "Сотовая связь"
+        static let hotspotImageName = "personalhotspot"
+        static let hotspotModePickerText = "Режим модема"
+        static let airTagImageName = "airtag.radiowaves.forward.fill"
+        static let vpnModeToggleText = "VPN"
+        static let bluetoothImageFrameParam = 25
+    }
+    
+    @State private var flightIsOn = false
+    @State private var VPNIsOn = false
+    @State private var wifiSection = 0
+    @State private var bluetoothSection = 0
+    @State private var cellularSection = 0
+    @State private var modemSection = 0
+    private var wifiParam = Constant.wifiParam
+    private var onOfParam = Constant.onOfParam
+    
+    
     var body: some View {
-        VStack {
-            Image(systemName: "globe")
-                .imageScale(.large)
-                .foregroundColor(.accentColor)
-            Text("Hello, world!")
+        NavigationView {
+            Form {
+                Section {
+                    setupProfilePicker
+                    Section {
+                        setupAppleOffersPicker
+                    }
+                }.navigationTitle(Text(Constant.settingText))
+                Section {
+                    setupNewIOSPicker
+                }
+                Section {
+                    setupFlightModeToggle
+                    setupWiFiPicker
+                    setupBluetoothPicker
+                    setupCellularPicker
+                    setupHotspotModePicker
+                    setupVPNPicker
+                }
+            }
+        }
+    }
+    
+    private var setupProfilePicker: some View {
+        return HStack {
```

Comment:
```suggestion
        HStack {
```
и ниже также убрать, где не требуется